relayer/pkg/chainlink/ocr2/medianreport/onchainconfig.go

```diff
@@ -32,40 +31,30 @@ func (codec OnchainConfigCodec) DecodeToFelts(b []byte) ([]*big.Int, error) {
 		return []*big.Int{}, fmt.Errorf("unexpected length of OnchainConfig, expected %v, got %v", length, len(b))
 	}
 
-	configVersion, err := bigbigendian.DeserializeSigned(byteWidth, b[:32])
-	if err != nil {
-		return []*big.Int{}, fmt.Errorf("unable to decode version: %s", err)
-	}
+	// convert from bytes to *big.Int
+	configVersion := new(big.Int).SetBytes(b[:32])
```

Comment:
We can get rid of the DeserializeSigned usage because we assume all reads from on-chain are unsigned